libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ChartConfigurationFlyout.tsx

```diff
@@ -183,16 +258,25 @@ export class ChartConfigurationFlyout extends React.Component<
     );
   }
 
-  // private getAllFeatureBasedCohorts() {
-  //   return this.props.featureBasedCohorts.map((_cohort, index) => {
-  //     return index;
-  //   });
-  // }
+  private onChoiceGroupChange = (
+    _ev?: React.FormEvent<HTMLElement | HTMLInputElement> | undefined,
+    option?: IChoiceGroupOption | undefined
+  ): void => {
+    if (option) {
+      if (option.key === this.datasetCohortsChoiceGroupOption) {
+        this.setState({ datasetCohortViewIsNewlySelected: true });

```

Comment:
datasetCohortViewIsNewlySelected is so long and a little confusing.  Why not just use an enum here?  Or perhaps call it:
datasetCohortViewSelected

Reply:
Because there are two:
- you can pass it into this component (via props)
- you can change the selection in this component (hence "newly" selected) but it won't take effect until you hit "confirm" so we have to store that on the state. That complicates things considerably but it's the only flow that's consistent with the UX we have elsewhere with flyouts.

Enum isn't needed since there are only 2 views and there will only ever be 2.

Reply:
I hope this was sufficient as an explanation. I don't think there's a particular action item to take so I'll close it. If not please reopen or message me and I'll address in a future PR. 🙂